x-pack/plugins/observability_ai_assistant/server/functions/query/index.ts

```diff
@@ -339,7 +340,8 @@ export function registerQueryFunction({
                 : {}),
             },
           };
-        })
+        }),
+        startWith(createFunctionResponseMessage({ name: 'query', content: { switch: true } }))
```

Comment:
What does `switch: true` mean?

Reply:
Nothing 🤷 the LLM needs _some_ properties, for whatever reason.